src/main/kotlin/no/nav/medlemskap/regler/common/Funksjoner.kt

```diff
@@ -36,6 +38,14 @@ object Funksjoner {
     infix fun List<Arbeidsforhold>.antallErIkke(tall: Int) = this.size != tall
 
     infix fun List<Int?>.finnesMindreEnn(tall: Int) = this.stream().anyMatch { p -> p == null || p < tall }
+
+    fun List<Medlemskap>.sammenhengendePerioder() = this.sorted().zipWithNext { a, b -> b.fraOgMed.isBefore(a.tilOgMed.plusDays(2)) }.all { it }
```

Comment:
Det er veldig kult at du har tatt kommentaren min om å bruke Kotlins støtte for mer funksjonell og deklarativ programmering til deg! Ulempen er at jeg ikke er dyktig nok til å forstå hva zipWithNext faktisk gjør.. Sånn sett er det godt at du har lagt til tester på dette, for da får jeg trygghet i at du har koda det riktig selv om jeg ikke helt forstår koden. (Jeg får sette meg ned og google zipWithNext for å forstå mer..)